tests/unit/delegate.ts

```diff
@@ -0,0 +1,93 @@
+import registerSuite = require('intern!object');
+import assert = require('intern/chai!assert');
+import { emit } from 'dojo-core/on';
+import { Handle } from 'dojo-core/interfaces';
+import delegate from 'src/delegate';
+import * as dom from 'src/dom';
+
+let container = document.createElement('div');
+let handles: (Handle)[] = [];
+
+document.body.appendChild(container);
```

Comment:
Argreed, because you might be surprised when some other test has already wiped your object off the DOM for you!